client/my-sites/plugins/plugin-action/test/index.jsx

```diff
@@ -1,67 +1,69 @@
 /**
  * External dependencies
  */
-var assert = require( 'assert' ),
-	ReactDom = require( 'react-dom' ),
-	React = require( 'react' ),
-	TestUtils = require( 'react-addons-test-utils' );
+import { expect } from 'chai';
+import mockery from 'mockery';
+import { mount, shallow } from 'enzyme';
+import React from 'react';
 
 /**
  * Internal dependencies
  */
-var PluginAction = require( '../plugin-action' );
-
-require( 'lib/react-test-env-setup' )();
+import EmptyComponent from 'test/helpers/react/empty-component';
+import useFakeDom from 'test/helpers/use-fake-dom';
+import useMockery from 'test/helpers/use-mockery';
 
 describe( 'PluginAction', function() {
-	afterEach( function() {
-		ReactDom.unmountComponentAtNode( document.body );
+	let PluginAction;
+
+	useFakeDom();
+	useMockery();
+
+	before( () => {
+		mockery.registerMock( 'components/info-popover', EmptyComponent );
+
+		PluginAction = require( '../plugin-action' );
 	} );
 
 	describe( 'rendering with form toggle', function() {
 		it( 'should have plugin-action class', function() {
-			var rendered = TestUtils.renderIntoDocument( <PluginAction /> ),
-				pluginAction = TestUtils.scryRenderedDOMComponentsWithClass( rendered, 'plugin-action' );
+			const wrapper = shallow( <PluginAction /> );
 
-			assert( 0 < pluginAction.length, 'a plugin action was rendered' );
+			expect( wrapper.find( '.plugin-action' ).length ).to.equal( 1 );
```

Comment:
I took it from Enzyme docs :P
![screen shot 2016-04-08 at 19 41 39](https://cloud.githubusercontent.com/assets/699132/14392511/f75985be-fdc1-11e5-999a-9705fbfd96d5.png)

Though they use your version there, too. I'll update :)